frontend/src/pages/Admin/Components/Faq/ManageFaq/ManageFaq.jsx

```diff
@@ -20,19 +20,27 @@ export function ManageFaq() {
   const [toastMessage, setToastMessage] = useState("");
   const [severity, setSeverity] = useState("success");
   const [reload, setReload] = useState(true);
+  const [editedFaq, setEditedFaq] = useState(null);
```

Comment:
why do we need to maintain the state for each of the fields? can we maintain the state of the faqObject? it would be much cleaner that way

Reply:
yes, i will make changes